tests/unit/test_constants.py

```diff
@@ -0,0 +1,11 @@
+from openscm import constants
+from datetime import timedelta
+
+
+def test_constants():
+    assert constants.MINUTE == timedelta(minutes=1).total_seconds()
+    assert constants.HOUR == timedelta(hours=1).total_seconds()
+    assert constants.DAY == timedelta(days=1).total_seconds()
+    assert constants.WEEK == timedelta(weeks=1).total_seconds()
+    assert constants.MONTH == timedelta(days=30).total_seconds()
+    assert constants.YEAR == timedelta(days=365).total_seconds()
```

Comment:
```suggestion
    
```

Surely we don't want to specify that a year is always 365 days yet? That also means that a year is no longer 12 months (given a month is 30 days). Having such internal conflicts already seems to be asking for trouble.

Reply:
As these are just convenienc constants for models using these specific constant time steps, I see no harm in them (aren’t you using them for Ph99 aswell? Or should they be defined by the particular adapter where needed? I’m not very emotional on this topic ;)

Reply:
> Or should they be defined by the particular adapter where needed?

This would be my preference to avoid confusion.

Reply:
all right